packages/docs/docs/compliance/clia-cap.md

```diff
@@ -52,3 +53,29 @@ Review and refer to the following items in preparation for your inspection.
+    - [ ] In this case the correct code for rejected specimen is `RI - Identification Problem`
+  - Test Order Accuracy: Percent of test orders correctly entered into a laboratory computer
+    - Though manual entry of orders is expected to be rare, the [QuestionnaireResponse](https://app.medplum.com/QuestionnaireResponse) resource is the source of truth for manual order entries.
+  - Specimen Acceptability: Percent of specimens received that are suitable for testing
+    - [ ] All [Specimens](https://app.medplum.com/Specimen)
+    - [ ] [Unsatisfactory Specimens](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,status&_offset=0&_sort=-_lastUpdated&status=unsatisfactory)
+  - Test Turnaround Time: Collection-to-reporting turnaround time or receipt-in- laboratory-to-reporting turnaround time of tests ordered. This may include orders of a “stat” priority (eg, emergency department or intensive care unit specimens), or routine priority, to include the percent of specimens with turnaround time that falls within an established limit (eg, the time that represents the 90th or 95th percentile of turnaround times or less than 30 minutes).
+    - [ ] Collection-to-reporting turnaround: [Specimen.collection.collectedDateTime](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,condition,collection&_offset=0&_sort=-_lastUpdated) to [DiagnosticReport.issued](https://app.medplum.com/DiagnosticReport?_count=20&_fields=id,_lastUpdated,subject,code,status,specimen,issued&_offset=0&_sort=-_lastUpdated)
+    - [ ] Receipt-in-laboratory-to-reporting turnaround: [Specimen.receivedTime](https://app.medplum.com/Specimen?_count=20&_fields=id,_lastUpdated,receivedTime&_offset=0&_sort=-_lastUpdated) to [DiagnosticReport.issued](https://app.medplum.com/DiagnosticReport?_count=20&_fields=id,_lastUpdated,subject,code,status,specimen,issued&_offset=0&_sort=-_lastUpdated)
+  - Critical Result Reporting: Percent of critical results with written record that results have been reported to caregivers; percent of critical results for which the primary clinician cannot be contacted in a reasonable period of time
+    - [ ] Patient contact are often implemented as [Communication](https://app.medplum.com/Communication?_count=20&_fields=id,_lastUpdated,status,subject,sender,payload&_offset=0&_sort=-_lastUpdated)
+    - [ ] Reason for outreach often represented in [CommunicationRequest.reasonReference](https://app.medplum.com/CommunicationRequest?_count=20&_fields=id,_lastUpdated,reasonReference&_offset=0&_sort=-_lastUpdated)
+    - [ ] [CommunicationRequests](https://app.medplum.com/CommunicationRequest?_count=20&_fields=id,_lastUpdated,reasonReference,status&_offset=0&_sort=-_lastUpdated&status:contains=completed) that have been completed.
```

Comment:
Similar to my comment above, we'll need to define an ontology for `CommunicationRequest.category` to filter out the ones meant for critical value logging.